examples/post_training_quantization/torch/ssd300_vgg16/main.py

```diff
@@ -14,6 +14,10 @@
 import subprocess
 from pathlib import Path
 
+# NNCF should be imported before import torch
```

Comment:
Why it should be?

Reply:
It should say

`# nncf.torch must be imported before torchvision`

Please see ticket 109653.

Reply:
Done